lib/gravity/gravity.go

```diff
@@ -17,70 +17,129 @@ limitations under the License.
 package gravity
 
 import (
-	"fmt"
+	"encoding/json"
+	"io/ioutil"
+	"os"
 	"os/exec"
 	"strings"
 
+	"github.com/alecthomas/template"
```

Comment:
Why do you need this? Seems like all it adds is elision of newlines which [is available](https://golang.org/doc/go1.6#template) since go1.6+